src/systems/eu/implementations/react/components/select/src/Select.jsx

```diff
@@ -111,6 +117,7 @@ const Select = ({
 };
 
 Select.propTypes = {
+  defaultValue: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
```

Comment:
I tried the following for array and none of them worked:

```jsx
defaultValue={['2', '3']}
```
```jsx
defaultValue={[2, 3]}
```

This worked well with setting France by default
```jsx
defaultValue="2"
```

I briefly checked [React docs](https://reactjs.org/docs/forms.html#the-select-tag) and found that when the select is multiple, you can set multiple values with `value` attribute which worked in our storybook example

```jsx
    <Select
      {...demoContentMultiple}
      value={['1', '2', '3']}
      id="select-multiple"
      options={demoContentMultiple.options}
      label={text('Label', demoContentMultiple.label)}
      helperText={text('Helper text', demoContentMultiple.helperText)}
      invalid={isInvalid}
      invalidText={text('Invalid text', demoContentMultiple.invalidText)}
      disabled={isDisabled}
      required={isRequired}
      requiredText={text('Required text', demoContentMultiple.requiredText)}
      optionalText={text('Optional text', demoContentMultiple.optionalText)}
      width={width}
      multiple
      data-ecl-select-default={defaultText}
      data-ecl-select-search={searchText}
      data-ecl-select-all={selectAllText}
    />
```

However, options are not containing any info regarding selected items 

![screenshot-nimbus-capture-2020 09 11-07_37_05](https://user-images.githubusercontent.com/1923476/92868221-c5450180-f401-11ea-8290-d7fddf6b6524.png)